ReelTalk/containers/Lolomo.js

```diff
@@ -24,12 +24,9 @@ var Lolomo = React.createClass({
 
   _showDetails: function(show) {
     this.props.navigator.push({
-      title: show.name,
+      title: 'Show',
```

Comment:
Fine for now, in the future I will probably get rid of/change this since everything is not necessarily a show